test/config.landlock.yaml

```diff
@@ -9,7 +9,7 @@ retrieve_databundle: # required to be "false" for nice CI test output
 countries: ["BW"]
 
 electricity:
-  renewable_carriers: [solar, onwind, hydro]
+  renewable_carriers: [solar, csp, onwind, hydro]
```

Comment:
@GbotemiB Did you have a specific idea when adding CSP only to a landlock test? (Looks good for me, just double-checking)

Reply:
At the point, CI was failing due to missing csp in the renewable carriers. Adding csp solved the CI issue.